knack/log.py

```diff
@@ -35,19 +32,27 @@ def get_logger(module_name=None):
 
 
 class _CustomStreamHandler(logging.StreamHandler):
+    COLOR_MAP = None
+
+    @classmethod
+    def get_color_wrapper(cls, level):
+        if not cls.COLOR_MAP:
+            import colorama
+
+            def _color_wrapper(color_marker):
+                def wrap_msg_with_color(msg):
+                    return color_marker + msg + colorama.Style.RESET_ALL
+                return wrap_msg_with_color
+
+            cls.COLOR_MAP = {
+                logging.CRITICAL: _color_wrapper(colorama.Fore.RED),
+                logging.ERROR: _color_wrapper(colorama.Fore.RED),
+                logging.WARNING: _color_wrapper(colorama.Fore.YELLOW),
+                logging.INFO: _color_wrapper(colorama.Fore.GREEN),
+                logging.DEBUG: _color_wrapper(colorama.Fore.CYAN)
+            }
 
-    def _color_wrapper(color_marker):  # pylint: disable=no-self-argument
-        def wrap_msg_with_color(msg):
-            return color_marker + msg + colorama.Style.RESET_ALL
-        return wrap_msg_with_color
-
-    COLOR_MAP = {
-        logging.CRITICAL: _color_wrapper(colorama.Fore.RED),
-        logging.ERROR: _color_wrapper(colorama.Fore.RED),
-        logging.WARNING: _color_wrapper(colorama.Fore.YELLOW),
-        logging.INFO: _color_wrapper(colorama.Fore.GREEN),
-        logging.DEBUG: _color_wrapper(colorama.Fore.CYAN)
-    }
+        return cls.COLOR_MAP.get(level, None)
```

Comment:
Can you add a test to `test_log.py` for this change?
As more and more products depend on knack, I want to make sure the test coverage is there for changes.

Reply:
ok.